core/src/autogluon/core/_setup_utils.py

```diff
@@ -27,12 +27,11 @@
     "networkx": ">=3.0,<4",  # Major version cap
     "tqdm": ">=4.38,<5",  # Major version cap
     "Pillow": ">=10.0.1,<12",  # Major version cap
-    "torch": ">=2.2,<2.5",  # Major version cap, sync with common/src/autogluon/common/utils/try_import.py
-    "lightning": ">=2.2,<2.4",  # Major version cap
-    "pytorch_lightning": ">=2.2,<2.4",  # Major version cap, capping `lightning` does not cap `pytorch_lightning`!
-    "async_timeout": ">=4.0,<5",  # Major version cap
-    "transformers[sentencepiece]": ">=4.38.0,<4.41.0",
-    "accelerate": ">=0.21.0,<0.34.0",
+    "torch": ">=2.2,<2.6",  # Major version cap, sync with common/src/autogluon/common/utils/try_import.py
+    "lightning": ">=2.2,<2.6",  # Major version cap
+    "async_timeout": ">=4.0,<6",  # Major version cap
+    "transformers[sentencepiece]": ">=4.38.0,<5",
+    "accelerate": ">=0.21.0,<1.2",
```

Comment:
Do we want to support two major versions of accelerate at the same time? I'm not so familiar with the package, so I don't know immediately how risky this is.

Reply:
Good point! Our accelerate package is quite outdated. I'm raising this PR to assess the potential issues with upgrading and to patch accordingly. 🛠️